src/server/game/Spells/Spell.cpp

```diff
@@ -5666,8 +5666,9 @@ SpellCastResult Spell::CheckCast(bool strict)
             }
             case SPELL_AURA_MOUNTED:
             {
-                if (m_caster->IsInWater())
-                    return SPELL_FAILED_ONLY_ABOVEWATER;
+                 // Fix for the Vashj'ir Mounts.
+                if (m_caster->IsInWater() && !(m_spellInfo->Id == 98718 || m_spellInfo->Id == 75207))
+					return SPELL_FAILED_ONLY_ABOVEWATER;
```

Comment:
gpn39f just now Member
wrong tabulator..
this will need additional pull necessary to correct that..